examples/nginx-app/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# nginx-centos7
+# Here you can use whatever image base is relevant for your application.
+FROM centos:centos7
+
+# Here you can specify the maintainer for the image that you're building
+MAINTAINER Victor Palade <ipalade@redhat.com>
+
+# Export the environment variable that provides information about the application.
+# Replace this with the according version for your application.
+ENV NGINX_VERSION=1.6.3
+
+# Set the labels that are used for Openshift to describe the builder image.
+LABEL io.k8s.description="Nginx Webserver" \
+    io.k8s.display-name="Nginx 1.6.3" \
+    io.openshift.expose-services="8080:http" \
+    io.openshift.tags="builder,webserver,html,nginx" \
+    io.openshift.s2i.scripts-url="image:///usr/libexec/s2i"
```

Comment:
comment on this particular label as being the one that tells s2i where to find the assemble/run/save-artifacts scripts.  it's the only one of these labels that is really critical.